packages/design-system/src/scalars/components/amount-field/amount-field.stories.tsx

```diff
@@ -176,11 +218,13 @@ export const HoverWithCurrency: Story = {
     label: "Enter Amount and Select Currency",
     name: "amount",
     allowedCurrencies: ["USD", "EUR"],
+    currencyPosition: "right",
     type: "AmountCurrency",
     value: {
       amount: 345,
       currency: "USD",
     },
+    step: 0,
```

Comment:
Why are we setting step 0 in all the stories? We should allow the default value on it to show the step showing the component as it is. If needed we can create a variant "without steps"

Reply:
Thank you for pointing this out! You are correct that the default value for step is 1, as documented in the component's documentation.

However, the reason for explicitly defining the step =0,  here is to align with the designs provided in Figma. In those designs, only one variant with a specific step value is represented, so we're making this explicit to ensure consistency with the design specifications.